token/services/vault/translator/action.go

```diff
@@ -22,17 +23,22 @@ type IssueAction interface {
 
 //go:generate counterfeiter -o mock/transfer_action.go -fake-name TransferAction . TransferAction
 
+// TransferAction is the action used to transfer tokens
 type TransferAction interface {
+	// Serialize returns the serialized version of the action
 	Serialize() ([]byte, error)
+	// NumOutputs returns the number of outputs of the action
 	NumOutputs() int
+	// GetSerializedOutputs returns the serialized outputs of the action
 	GetSerializedOutputs() ([][]byte, error)
+	// IsRedeemAt returns true if the output is a redeem output at the passed index
 	IsRedeemAt(index int) bool
+	// SerializeOutputAt returns the serialized output at the passed index
 	SerializeOutputAt(index int) ([]byte, error)
+	// GetInputs returns the inputs's identifiers of the action
```

Comment:
returns the identifiers of the inputs in the action.

Reply:
okay